src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php

```diff
@@ -38,7 +38,7 @@ public function createService(Locator $serviceLocator)
         $debugbar->addCollector(new ConfigCollector($appConfig, 'ApplicationConfig'));
 
         // Db profiler
-        if ($serviceLocator->has($dbServiceName) && isset($config['db']['driver'])) {
```

Comment:
Hm, ok! To start, I will try simulate exception (#5) via unit-tests and fix this code section with check. Later I notify you.